dashboard/src/features/accessControl/components/EditPolicies.jsx

```diff
@@ -15,10 +15,14 @@ class EditPolicies extends React.Component {
     return(
       <div className={styles.main}>
         {policyOptions.map(option => {
+          const grant = this.props.item.grants.find(g => g.policy == option.value)
```

Comment:
Based on https://github.com/chain/chain/pull/1085#issuecomment-298082163 I expected that one couldn't create a non-protected version of a protected grant, nor would two ever co-exist. 

I'd like to hold off on that change until some protected grants have landed? It's really hard to test these right now and get a sense of what the right experience is.